mods/ts/rules/vehicles.yaml

```diff
@@ -837,4 +628,3 @@ STNK:
 	RenderVoxels:
 	WithVoxelBody:
 	-MustBeDestroyed:
-
```

Comment:
There were actually two newlines at the end of this file.  This removal brings us back to the standard one.